apps/wallet-mobile/src/features/Initialization/TermsOfServiceChangedScreen/TermsOfServiceChangedScreen.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react'
+import {defineMessages, useIntl} from 'react-intl'
+import {StyleSheet, Text, TouchableOpacity} from 'react-native'
+import {SafeAreaView} from 'react-native-safe-area-context'
+
+import {Button, Spacer, YoroiLogo} from '../../../components'
+import {BlueCheckbox} from '../../../components/BlueCheckbox'
+import {COLORS} from '../../../theme'
+import {useAgreeWithTermsOfService,useNavigateTo} from '../common'
+
+export const TermsOfServiceChangedScreen = () => {
+  const [tosAccepted, setTosAccepted] = React.useState(false)
+  const {agree} = useAgreeWithTermsOfService()
+
+  const onPressContinue = () => {
+    agree()
+  }
+
+  const onPressTosCheckbox = () => {
+    setTosAccepted((checked) => !checked)
+  }
+
+  const navigateTo = useNavigateTo()
+  const strings = useStrings()
+
+  const onPressLink = () => {
+    navigateTo.accepTos()
+  }
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <YoroiLogo />
+
+      <Spacer height={80} />
+
+      <Text style={styles.title}>{strings.title}</Text>
+
+      <Spacer height={24} />
+
+      <Text style={styles.description}>{strings.description}</Text>
+
+      <Spacer height={24} />
+
+      <BlueCheckbox checked={tosAccepted} onPress={onPressTosCheckbox}>
+        <Text style={styles.checkboxText}>{`${strings.tosIAgreeWith} `}</Text>
+
+        <TouchableOpacity onPress={onPressLink}>
+          <Text style={[styles.checkboxText, styles.checkboxLink]}>{strings.tosAgreement}</Text>
+        </TouchableOpacity>
+      </BlueCheckbox>
+
+      <Spacer fill />
+
+      <Button title="continue" shelleyTheme disabled={!tosAccepted} onPress={onPressContinue} />
```

Comment:
All CTAs should be sticky at bottom

Reply:
also `continue` needs to be translated

(I need to fix this in my pr too)